posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md

```diff
@@ -0,0 +1,457 @@
+---
+authors:
+- Dennis Sharpe
+tags:
+- JHipster
+- JHipster Lite
+- React
+- Angular
+- Hexagonal Architecture
+date: 2022-12-13T12:21:50.000Z
+title: "How to Create a Quick React and Angular App with JHipster Lite"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/12/jhipster_lite_logo.png
+---
+# What is JHipster Lite?
+
+JHipster Lite is an application generator that generates step-by-step only what you need for a modern web application and/or microservice architecture. As you might imagine from the name, JHipster Lite is a different approach to application generation than its predecessor, [JHipster](https://www.jhipster.tech/).
```

Comment:
Possible link opportunity for "application generator" for those who may never heard of one: https://www.pcmag.com/encyclopedia/term/application-generator